oxidation/bindings/web/.gitignore

```diff
@@ -0,0 +1 @@
+pkg/
```

Comment:
Good point to put this in a local .gitignore in order not to pollute the (big enough) root .gitignore 👍